src/main/java/tornadofx/Properties.kt

```diff
@@ -170,4 +170,34 @@ private class UnsynchronizedSingleAssign<T> : SingleAssign<T> {
     }
 
     override fun isInitialized() = initialized
-}
\ No newline at end of file
+}
+
+operator fun <T> Property<T>.getValue(thisRef: Any?, property: KProperty<*>): T = value
+operator fun <T> Property<T>.setValue(thisRef: Any?, property: KProperty<*>, value: T) {
+    this.value = value
+}
+
+operator fun DoubleProperty.getValue(thisRef: Any?, property: KProperty<*>) = value!!
+operator fun DoubleProperty.setValue(thisRef: Any?, property: KProperty<*>, value: Double) {
+    this.value = value
+}
+
+operator fun FloatProperty.getValue(thisRef: Any?, property: KProperty<*>) = value!!
+operator fun FloatProperty.setValue(thisRef: Any?, property: KProperty<*>, value: Float) {
+    this.value = value
+}
+
+operator fun LongProperty.getValue(thisRef: Any?, property: KProperty<*>) = value!!
+operator fun LongProperty.setValue(thisRef: Any?, property: KProperty<*>, value: Long) {
+    this.value = value
+}
+
+operator fun IntegerProperty.getValue(thisRef: Any?, property: KProperty<*>) = value!!
```

Comment:
It returns `Int`. The 5 primitive properties (`DoubleProperty`, `FloatProperty`, `LongProperty`, `IntegerProperty`, and `BooleanProperty`) protect against nulls, which is why they needed special cases. If you try to set them to `null`, it is stored as 0 or `false`.